### PR TITLE
Beau fyupdates

### PR DESCRIPTION
**Pull request by beaudjpr**
Adding accessibility features, and applying other minor fixes

### CODE REVIEW

File: inst/docs/landing.Rmd [hunk 1]
```diff
@@ -428,52 +429,52 @@ growth_chart
...
                                    business_banner,
                                    business_charts,
                                    business_summary)
 
-austourism::render_object_html(object = business_landing_charts,
-                               title = 'Creative-Business Charts',
+austourism::render_object_html(object = landing_business,
+                               title = 'Creative Business Overview',
                                css = 'style.css',
                                output_file = here::here(IFRAME_FOLD,
```
Comment (mrjoh3) [inline inst/docs/landing.Rmd:453]: didn't we update this variable name? Variable names like this should also be defined in `setup.Rmd` as they are the same for each file. It is best to only define these variables in one place.

File: inst/docs/setup.Rmd [hunk 2]
```diff
@@ -33,6 +35,8 @@ shinyWidgets::useBs4Dash()
 
 FOLDER <- here::here('inst', 'quarterly_update', 'Outputs')
 FOLDER_RMD <- here::here('inst', 'docs')
+IFRAME_FOLD <- here::here('docs') # OUTPUT_FOLDER
```
Comment (mrjoh3) [inline inst/docs/setup.Rmd:38]: sorry, I know I'm being pedantic, but iframes are only in the host website. This folder holds content that is consumed by the iframes. Please update to `OUTPUT_FOLDER` or `CONTENT_FOLDER` and make sure this variable is only defined here,

File: inst/docs/setup.Rmd [hunk 3]
```diff
@@ -124,7 +129,7 @@ for (rmd in rmds) {
   }
   
   rmarkdown::render(here('inst', 'docs', rmd),
-                    output_file = outfile,
+                    output_file = glue('test_{outfile}'),
```
Comment (mrjoh3) [inline inst/docs/setup.Rmd:132]: why have we added `test_` it is not needed

**Review by mrjoh3 — CHANGES_REQUESTED** [review]
main things are to remove the `quick_iframe` functionfrom each file and make sure the output files aren't called `test` or `version`. Can you also update the rmd file names. there is no longer any reason to include `version2` in the file names.

**Review by Henry-DJPR — APPROVED** [review]

**Pull request merged**